### DISCUSSION TITLE
Evolution, Liferea and Gnome-Logs problem

### DISCUSSION THREAD

**Discussion by Froggy232** [netblue30/firejail#4112] [Q&A] [answered]
Hi,
So, I just installed firejail and I have one pretty important problem, and two little :

1) I installed firejail along with apparmor on archlinux-hardened, then executed `sudo firecfg`. It created symlinks for all apps, except evolution, I don't know why. It didn't wrote anything about evolution. If I try to launch evolution from a terminal or gnome, it obviously start without firejail. But if I type `firejail evolution` in a terminal, it work great but start with a new configuration.
How can I add a symlink for evolution? Why it isn't the case by default, is there a known issue? And how is it possible to make it detect my actual profile?

2) Liferea work great, but the preview panel doesn't work. When I click on a new, the panel stay empty and I need to open the link in firefox to read it. Also, it open a virgin firefox profile, inside the liferea sandbox.
Is there a possibility to make the preview panel work? Also, is there a possibility to open the link in the firefox sandbox/profile?

3) When I use gnome-logs with firejail, it can't access any logs. Is it possible to whitelist something to solve the problem? I use a tmpfs on /var/log, I don't know if there is a link.

Thanks you a lot,
Best regards

**Comment by rusty-snake** [comment, marked as answer]
1. #3647
2. #3647
3. https://gitlab.com/rusty-snake/kyst/-/blob/ccc67b8c9b26cde7734643b3e2a7c4558c11f766/firejail/gnome-logs.local should be added upstream. [EDIT: done]

So 1. and 2. are the same issue. You can not use firejail from the ATM :cry: .
<sup><sub>... or you use`env WEBKIT_FORCE_SANDBOX=0` :speak_no_evil: :see_no_evil: :hear_no_evil:. **DON'T DO SO** </sub></sup>

**Reply by Froggy232** [reply]
Thanks you a lot, I really can't stress it enough.
Is it possible to make you a donation?
